ci-operator/step-registry/aws/provision/bastionhost/post/aws-provision-bastionhost-post-ref.yaml

```diff
@@ -0,0 +1,22 @@
+ref:
+  as: aws-provision-bastionhost-post
```

Comment:
how about merging the step to `aws-provision-bastionhost`, considering private cluster install also need the ssh SG when collecting the must gather log for private clusters.

Reply:
@jianlinliu The problem is the worker SG would be generated during the cluster installation, so we don't know the SG ID yet when running `aws-provision-bastionhost `

> considering private cluster install also need the ssh SG when collecting the must gather log for private clusters
yes, just also for this, so I added this standalone step instead of adding this in RHEL worker steps, so we can insert it for necessary private cluster jobs.

Reply:
> The problem is the worker SG would be generated during the cluster installation, so we don't know the SG ID yet

Ah, got it.